.github/workflows/node.js.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Install pnpm
         uses: pnpm/action-setup@v1.2.1
         with:
-          version: 5.18.9
+          version: 5.18.8
```

Comment:
in the future i think this should be a matrix build across versions of node and operating systems.

i can show you how to do this if you don't already know; it's easy.

i don't think this PR should block on this, so [I opened a new issue about it](https://github.com/peggyjs/peggy/issues/75) instead.